cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -159,6 +164,15 @@ function update_addons() {
   fi
 }
 
+function ensure_addons() {
+  # Create objects already exist should fail.
+  # Filter out `AlreadyExists` message to not noisily log.
+  ${KUBECTL} ${KUBECTL_OPTS} create --namespace=${SYSTEM_NAMESPACE} -f ${ADDON_PATH} -l ${SERVICE_ENSURE_EXIST_LABEL} \
```

Comment:
Also bad news on this one. I tried implement `--ignore-already-exists` for `kubectl create` but encountered another problem: `kubectl create` returns error when there is no object passed to create. With this flag enabled, if we pass in a few objects to create but all of them exist, we still get an error because the "nothing create error".

I need to put more thoughts on it. But feel like this could be done later.